workflow_scripts/install_linux_proto.sh

```diff
@@ -0,0 +1,14 @@
+yum -y install wget
+
+export NUMCORES=`grep -c ^processor /proc/cpuinfo`
+
+pb_dir="./cache/pb"
+PB_VERSION=2.6.1
```

Comment:
Is there a centralized place to manage the protobuf versions we are using? In other words, are we using 2.6.1 in all the places?